Lib/fontTools/varLib/__init__.py

```diff
@@ -81,7 +79,10 @@ def _add_fvar(font, axes, instances):
 		coordinates = instance.location
 
 		if "en" not in instance.localisedStyleName:
-			assert instance.styleName
+			if not instance.styleName:
+				raise VarLibValidationError(
+					f"Instance at location '{coordinates}' must have a styleName."
```

Comment:
instance must have a styleName ... or an 'en' localizedStyleName (basically the default english style name is required)